hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/conf/ConfServlet.java

```diff
@@ -43,6 +43,7 @@ public class ConfServlet extends HttpServlet {
   protected static final String FORMAT_JSON = "json";
   protected static final String FORMAT_XML = "xml";
 
+
```

Comment:
Avoid

Reply:
Thank you very much for your help reviewing the code, I will fix it.